docs/config.md

```diff
@@ -129,12 +129,12 @@ change_id = "#ff1525"
 ```
 
 If you use a string value for a color, as in the examples above, it will be used
-for the foreground color. You can also set the background color, or make the
-text bold, italic, or underlined. For that, you need to use a table:
+for the foreground color. You can also set the background color, reverse colors,
+or make the text bold, italic, or underlined. For that, you need to use a table:
 
 ```toml
 [colors]
-commit_id = { fg = "green", bg = "#ff1525", bold = true, italic = true, underline = true }
+commit_id = { fg = "green", bg = "#ff1525", reverse = true, bold = true, italic = true, underline = true }
```

Comment:
nit: might be better to split the example keys. It wouldn't make sense to set fg/bg and reverse them.

Reply:
I've included a new key. I also moved one other flag, just so neither gets too crowded.